data/base/sle/sle15/sp5/packages.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  _namespace_sle_cnf:
+    package:
+      - scout-command-not-found
```

Comment:
This is actually a packaging bug. The new package should have a `Provides: command-not-found` such that this change is not necessary.

Reply:
`scout-command-not-found` has that Provides, but it appears OBS prefers package name matches to provides matches, even if the direct match is from a lower prio repo (a bit debatable IMO).